zilencer/urls.py

```diff
@@ -22,8 +23,9 @@
     rest_path("remotes/push/unregister", POST=unregister_remote_push_device),
     rest_path("remotes/push/unregister/all", POST=unregister_all_remote_push_devices),
     rest_path("remotes/push/notify", POST=remote_server_notify_push),
-    # Push signup doesn't use the REST API, since there's no auth.
+    # Push signup or deactivation doesn't use the REST API, since there's no auth.
     path("remotes/server/register", register_remote_server),
+    path("remotes/server/deactivate", deactivate_remote_server),
```

Comment:
I think this should use the `rest_path` authentication, just like sending a push notification -- the `zulip_org_id / zulig_org_key` pair is authentication.